implementation/src/main/java/io/smallrye/mutiny/operators/multi/MultiOnItemInvoke.java

```diff
@@ -0,0 +1,44 @@
+package io.smallrye.mutiny.operators.multi;
+
+import static io.smallrye.mutiny.helpers.ParameterValidation.nonNull;
+
+import java.util.function.Consumer;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.helpers.Subscriptions;
+import io.smallrye.mutiny.subscription.MultiSubscriber;
+
+public class MultiOnItemInvoke<T> extends AbstractMultiOperator<T, T> {
```

Comment:
Make sure with have the associated TCK.

Reply:
We already had `MultiOnItemInvokeTckTest`